applications/ShapeOptimizationApplication/custom_utilities/mapping/mapper_vertex_morphing.h

```diff
@@ -511,7 +511,7 @@ class MapperVertexMorphing
         for(auto& node_i : mrDesignSurface.Nodes())
         {
             array_3d& coord = node_i.Coordinates();
-            sumOfAllCoordinates += coord[0] + coord[1] + coord[2];
+            sumOfAllCoordinates += std::abs(coord[0]) + std::abs(coord[1]) + std::abs(coord[2]);
```

Comment:
Is there a method to calculate the norm of the coordinates array?

Reply:
I thought it makes sense here not to use quadratic values (l2 norm) since then the values might become very large and this is just a sensor value anyway.